test/parallel/test-child-process-exec-stdio-inherit.js

```diff
@@ -0,0 +1,46 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
All of this copyright text should not be included in new tests.

Reply:
Fixed, thanks